packages/widget/src/state/skipClient.ts

```diff
@@ -25,11 +25,15 @@ export const skipClientConfigAtom = atom<SkipClientOptions>({
 
 export const themeAtom = atom<Theme>(defaultTheme);
 
+export const solanaWalletsAtom = atom<Wallet[]>();
```

Comment:
I don't think this no need to be an atom, can you elaborate why we need this to be an atom?

Reply:
main issue is we can't get the wallets from the context of `SolanaProvider` in jotai besides doing something like this

Reply:
the alternative obviously would be to just stop using the SolanaProvider

Reply:
let's stop using SolanaProvider and revert to what we were doing before. too op if we using only `useWallet` hooks from the solana react package. because SolanaProvider or context is useless here